README.md

```diff
@@ -273,15 +257,15 @@ I would like to thank...
 
 - **[Question_2]**
 
-  - [Answer_2]
+  - [Answer_2] -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Hey, please, let's remove this link as its currently duplicated.

Reply:
@zieeco I  have removed this duplicated part